src/sentry/db/models/base.py

```diff
@@ -54,6 +55,23 @@ def __reduce__(self):
     def __setstate__(self, state):
         self.__dict__.update(state)
 
+    def set_cached_field_value(self, field_name, value):
```

Comment:
This could do with a comment explaining what this is for. Maybe clarify that this is a local cache, is only intended for relations to other models, etc.